internal/restic/node_windows.go

```diff
@@ -22,18 +27,104 @@ func (node Node) restoreSymlinkTimestamps(path string, utimes [2]syscall.Timespe
 
 // Getxattr retrieves extended attribute data associated with path.
 func Getxattr(path, name string) ([]byte, error) {
-	return nil, nil
+	switch name {
+	case "CreationTime":
+		fileinfo, err := os.Stat(path)
+		if err != nil {
+			return nil, errors.Wrap(err, "Getxattr")
+		}
+
+		s, ok := fileinfo.Sys().(*syscall.Win32FileAttributeData)
+		if ok && s != nil {
+			var creationTime bytes.Buffer
+			enc := gob.NewEncoder(&creationTime)
+			
+			if err := enc.Encode(syscall.NsecToTimespec(s.CreationTime.Nanoseconds())); err != nil {
+				return nil, errors.Wrap(err, "Getxattr")
+			}
+			return creationTime.Bytes(), nil
```

Comment:
I don' think you need all this conversions and encoding. Simple `binary` conversion is enough:
```          
var creationTime [8]byte
binary.LittleEndian.PutUint32(creationTime[0:4], s.CreationTime.LowDateTime)
binary.LittleEndian.PutUint32(creationTime[4:8], s.CreationTime.HighDateTime)
return creationTime[:], nil
```